CMakeLists.txt

```diff
@@ -701,7 +703,7 @@ set(C_SRCS
   src/witness.c
 )
 
-if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+if(CMAKE_SYSTEM_NAME MATCHES "Darwin" AND NOT disable-zone-allocator)
```

Comment:
I don't know how to get apple zone, so temporary disable this according to configure.ac.